backend/fms_core/models/_constants.py

```diff
@@ -3,6 +3,9 @@
 STANDARD_NAME_FIELD_LENGTH = 200
 STANDARD_SEQUENCE_FIELD_LENGTH = 500
 PROJECT_STATUS_CHOICES = ["Open", "Closed"]
+STRANDEDNESS_CHOICES = ["Double stranded", "Single stranded"]
```

Comment:
You should reference the constants in here so if we change them it propagates here too.